tests/test_permissions.py

```diff
@@ -25,17 +25,29 @@
 
 
 # registration is done in setUp
-@require_permission(permissions.OVERSEER)
-async def restricted(context):
-    await context.reply("Restricted command was executed.")
+
+
+# @require_permission(permissions.OVERSEER)
+# async def restricted(context):
+#     await context.reply("Restricted command was executed.")
```

Comment:
Is this likely to be needed again soon? Commented-out code can get confusing quickly imo.

Reply:
Il have a look. good point.